common/lib/dependabot/update_checkers/base.rb

```diff
@@ -136,7 +136,7 @@ def latest_resolvable_version
 
       # Lowest available security fix version not checking resolvability
       # @return [Dependabot::<package manager>::Version, #to_s] version class
-      sig { overridable.returns(Dependabot::Version) }
+      sig { overridable.returns(T.nilable(Dependabot::Version)) }
```

Comment:
🔎 What do you think of making this `nilable` in the [Nuget update checker](https://github.com/dependabot/dependabot-core/blob/09263a64947d4762c0e69a6d913b8a7683d489bb/nuget/lib/dependabot/nuget/update_checker.rb#L40-L43)? There's an `override` there, which expects a `Dependabot::Version` to be returned.

Reply:
Thank you; addressed it in the file you recommended.

Reply:
Does it need to be addressed in the base class and all subclasses?